worker/s3caller/package_test.go

```diff
@@ -0,0 +1,38 @@
+// Copyright 2023 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package s3caller
+
+import (
+	"testing"
+
+	"go.uber.org/mock/gomock"
+	gc "gopkg.in/check.v1"
+
+	jujujujutesting "github.com/juju/juju/testing"
```

Comment:
Probably best to rename this `coretesting`, as it is called in a few other places

Reply:
done!